lib/conversion.py

```diff
@@ -0,0 +1,11 @@
+# TODO (2025-04-03) Test, complete
+
+def convert_time(time, source, target):
+    if source == target:
+        return time
+    elif source == 'h' and target == 's':
+        return round(float(time) * 3600)
+    elif source == 'm' and target == 's':
+        return round(float(time) * 60)
+    else:
+        raise ValueError(f"Conversion from {source} to {target} unsupported for now")
```

Comment:
This is incomplete, but it works for a demo. It needs to first convert down to seconds, then up to whatever it needs.